examples/progressbar_simpletest.py

```diff
@@ -27,10 +27,10 @@
 
 current_progress = 0.0
 while True:
-    while current_progress <= 1.0:
-        print("Progress: {}%".format(current_progress * 100))
-        progress_bar.progress = current_progress
-        current_progress += 0.05
-        if current_progress >= 1.0:
-            current_progress = 0.0
-        time.sleep(0.01)
+    for current_progress in range(0, 21):
+        print("Progress: {}%".format(current_progress * 0.05))
```

Comment:
This outputs 0 -> 1%. I changed it to
`        print("Progress: {}%".format(current_progress * 0.05 *100))
`

Reply:
Good catch thank you. I'll push a new commit to fix this soon.